samples/dashboard/demos/component-highcharts/demo.js

```diff
@@ -0,0 +1,20 @@
+const dashboard = new Dashboard.Dashboard('container', {
+    gui: {
+        layouts: [{
+            rows: [{
+                cells: [{
+                    id: 'cell-id'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        type: 'Highcharts',
```

Comment:
Can you add two cells with two different charts so it looks more like a dashboard?